linetools/lists/mk_sets.py

```diff
@@ -0,0 +1,125 @@
+"""
+Module for making set lists
+  INTERNAL USE ONLY
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import os, imp, glob
+
+from astropy import units as u
+from astropy.units.quantity import Quantity
+from astropy import constants as const
+from astropy.io import fits, ascii
+from astropy.table import QTable, Column, Table
+
+from xastropy.xutils import xdebug as xdb
+
+lt_path = imp.find_module('linetools')[1]
+xa_path = imp.find_module('xastropy')[1]
+
+#
+def mk_ism(outfil=None, clobber=False):
+    ''' Make the ISM list from grb.lst in xastropy
```

Comment:
Ok.  I’ve seen both so far in Python
but am fine with overwrite.

X

On Jun 21, 2015, at 1:15 PM, Neil Crighton <notifications@github.com> wrote:

> In linetools/lists/mk_sets.py:
> 
> > +import os, imp, glob
> > +
> > +from astropy import units as u
> > +from astropy.units.quantity import Quantity
> > +from astropy import constants as const
> > +from astropy.io import fits, ascii
> > +from astropy.table import QTable, Column, Table
> > +
> > +from xastropy.xutils import xdebug as xdb
> > +
> > +lt_path = imp.find_module('linetools')[1]
> > +xa_path = imp.find_module('xastropy')[1]
> > +
> > +#
> > +def mk_ism(outfil=None, clobber=False):
> > +    ''' Make the ISM list from grb.lst in xastropy
> Can we use 'overwrite' instead of 'clobber'? It's less confusing for the uninitiated.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> !DSPAM:826,55871bda120569428521!

----------------------------------------------
J. Xavier Prochaska
UCO/Lick Observatory
1156 High St.
UC Santa Cruz
Santa Cruz, CA 95064

http://www.ucolick.org/~xavier/
http://imps.ucolick.org/
http://www.qpqsurvey.org/
http://k1dm3.ucolick.org/
http://www.ucolick.org/~xavier/IDL/index.html
https://github.com/profxj/xastropy

831-459-2135 (Direct)
831-459-2991 (UCO/Lick Main)
831-459-5244 (Fax)